spec/requests/api/v1/accounts_show_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'GET /api/v1/accounts/{account_id}' do
+  it 'returns account entity as 200 OK' do
+    account = Fabricate(:account)
+
+    get "/api/v1/accounts/#{account.id}"
+    response_body = body_as_json
```

Comment:
You could probably skip the local var assignment here and just expect directly on the `body_as_json` method (or on rails built-in `parsed_body` method...)

Reply:
fixed.
In my initial draft, I asserted all key value pairs.
However, it became too long.
So, I reduced assertions to only id.
Assignment body_as_json to response_body local variable is a trace of it.

I prefer to symbolize keys when asserting the response body.